keps/349-all-or-nothing/README.md

```diff
@@ -225,6 +246,28 @@ type WaitForPodsReady struct {
 	// RequeuingStrategy defines the strategy for requeuing a Workload.
 	// +optional
 	RequeuingStrategy *RequeuingStrategy `json:"requeuingStrategy,omitempty"`
+
+	// WaitForAdmissionChecks defines configuration for the WaitForAdmissionChecks feature,
+	// which is used to ensure that specific AdmissionCheck is ready within the specified time.
+	// If there is no specified configuration for an AdmissionCheck, there is no timeout for it.
+	// +listType=map
+	WaitForAdmissionChecks []WaitForAdmissionChecks
+}
+
+type WaitForAdmissionChecks struct {
+	// Name of the AdmissionCheck the configuration refers to
+	AdmissionCheck string
+
+	// Timeout defines the time for workload with reserved quota to reach the
+	// Admitted=true condition. When the timeout is exceeded, the workload
```

Comment:
Is this timeout really for the `Admitted=true`, or individually per AdmissionCheck? 

If we consider the timeouts individually we should rather check on the LastTransitionTime of the AdmissionChecksState (Pending -> Ready).

If we put timeout on all Admission Checks, then we could look for LastTransitionTime of QuotaReserved and Admitted.

I think both approaches are valid, but we need to decide on one, and make it explicit.

Reply:
Good point, my idea was to have a configuration per AdmissionCheck, hence the timeout should be based on the AdmissionCheckStates.

Reply:
Done